core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/common/Status.java

```diff
@@ -49,4 +49,24 @@ public boolean isAvailable() {
     public boolean isInProgress() {
         return getStatusKind().equals(StatusKind.PROGRESS);
     }
+
+    public boolean isStopped() {
+        return STOPPED == this;
+    }
+
+    public boolean isStartState() {
+        return Status.AVAILABLE.equals(this)
+                || UPDATE_IN_PROGRESS.equals(this)
+                || Status.START_FAILED.equals(this)
+                || Status.START_REQUESTED.equals(this)
+                || Status.START_IN_PROGRESS.equals(this);
+    }
+
+    public boolean isStopState() {
+        return Status.STOPPED.equals(this)
+                || UPDATE_IN_PROGRESS.equals(this)
+                || Status.STOP_IN_PROGRESS.equals(this)
+                || Status.STOP_REQUESTED.equals(this)
+                || Status.STOP_FAILED.equals(this);
```

Comment:
Wouldn't this basically the same as NOT started and NOT stopped?
And something like stoppable and startable?